src/codegen/codegen_c_visitor.hpp

```diff
@@ -1656,6 +1656,13 @@ class CodegenCVisitor: public visitor::ConstAstVisitor {
         , float_type(float_type)
         , optimize_ionvar_copies(optimize_ionvar_copies) {}
 
+    static int count_length(const std::vector<SymbolType>& variables) {
+        int length = 0;
+        for (const auto& variable: variables) {
+            length += variable->get_length();
+        }
+        return length;
+    };
```

Comment:
this function is not really needed since it can be replaced with a one-liner:

```
std::accumulate(v.begin(), v.end(), 0, [](int l, const SymbolType& variable) {return a+= variable->get_length(); })
```